proto/schema.proto

```diff
@@ -0,0 +1,170 @@
+syntax="proto3";
+
+option go_package = "github.com/kubernetes-csi/external-snapshot-metadata/pkg/grpc";
```

Comment:
I guess we don't need to import any other defintions. The CSI spec has 
```
import "google/protobuf/descriptor.proto";
import "google/protobuf/timestamp.proto";
import "google/protobuf/wrappers.proto";

option go_package = 
  "github.com/container-storage-interface/spec/lib/go/csi";
```

Reply:
That's right